src/DiffOp/Diff/Diff.php

```diff
@@ -145,21 +145,23 @@ private function preSetElement( $index, DiffOp $value ): bool {
 	 *
 	 * @param string $serialization
 	 */
-	public function unserialize( $serialization ) {
-		$serializationData = unserialize( $serialization );
+	public function unserialize( $serialization ): void {
+		$this->__unserialize( unserialize ($serialization) );
+	}
 
-		foreach ( $serializationData['data'] as $offset => $value ) {
+	public function __unserialize( array $data ): void {
```

Comment:
Probably want to copy docs across

Reply:
I don’t think copying `@see Serializable::unserialize` for an `__unserialize` method makes much sense, it’s not the same method…